hack/Makefile

```
@@ -61,4 +64,4 @@ generate-go:
 # because the latter copies around files that need to be generated by the
 # former. Without doing it in this order, the `generate` target will not be
 # idempotent and would have to be executed twice to achieve the desired result.
-generate: generate-from-swagger generate-go generate-events generate-mocks generate-configuration generate-bundle
+generate: generate-from-swagger generate-go generate-events generate-mocks generate-configuration generate-test-versions generate-bundle
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)